meta/SchemaBuilder.php

```diff
@@ -140,8 +142,8 @@ protected function fixLabel($wantedlabel, &$labels) {
     protected function newSchema() {
         if(!$this->time) $this->time = time();
 
-        $sql = "INSERT INTO schemas (tbl, ts) VALUES (?, ?)";
-        $this->sqlite->query($sql, $this->table, $this->time);
+        $sql = "INSERT INTO schemas (tbl, ts, user) VALUES (?, ?, ?)";
+        $this->sqlite->query($sql, $this->table, $this->time, blank($this->user) ? $_SERVER['REMOTE_USER'] : $this->user);
```

Comment:
I would initialize $this->user in the constructor, then you won't need this ugly construct.